test/openAPI/features/wallet_linked_authenticate.feature

```diff
@@ -0,0 +1,47 @@
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should contain linkedTransactionId
+    And The /linked-authorization/authenticate endpoint response should match json schema without errors
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of an invalid linkedTransactionId
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/authenticate request with given invalid linkedTransactionId
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_transaction_id"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of an invalid individualId
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/authenticate request with given invalid individualId
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_identifier"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of an invalid challengeList
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/authenticate request with given invalid challengeList
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_no_of_challenges"
```

Comment:
Can we add a test case for re authenticating using the same linked transaction id for the second time and expect invalid transaction id ?

Reply:
I'm afraid there is no possible to do it  - the issue is with the mocks, we do not have server where we can keep status of the transaction id and manage it

Reply:
We don't have to maintain a state for this, we can use the same link transaction id and call authenticate endpoint twice and expect error for the second request

Reply:
this is taken care in the new PR ?